eQuiz/modules/eQuiz.Web/Areas/Admin/Scripts/review/equiz.startFromFilter.js

```diff
@@ -0,0 +1,9 @@
+﻿(function(angular) {

```

Comment:
Rename the file according to https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#naming